### DISCUSSION TITLE
What is the purpose of 3 package.json files

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [c9fe/22120#104] [Q&A] [answered]
Hi,
I noticed there is 3 package.jsons for Diskernet. I didn't see any instructions on how to utilize each one of them & when we might want to do so.

@i5ik - would you mind explaining more how/when to use each?

**Comment by crisdosaygo** [comment, marked as answer]
Thanks @cliffordfajardo I can see you are a very talented developer, thanks for joining the discussion!

Well spotted on this question.

The two package.json files are alternately for: 1) building and distribution (to NPM) and 2) development.

`"type": "module"` will not work with building with webpack and sending to NPM in this project, but I like to use it for development.

Thanks for this question :)
